src/gui/mzroll/eicwidget.cpp

```diff
@@ -1994,35 +1984,39 @@ void EicWidget::timerEvent(QTimerEvent * event) {
 	}
 }
 
-/*
-@author: Sahil
-*/
-//TODO: Sahil, Added this function while merging eicwidget
-void EicWidget::addMS2Events(float mzmin, float mzmax) {
-
+void EicWidget::addMS2Events(float mzmin, float mzmax)
+{
     qDebug() << "addMS2Events() " << mzmin << " " << mzmax;
 
     MainWindow* mw = getMainWindow();
     vector <mzSample*> samples = mw->getVisibleSamples();
 
     if (samples.size() <= 0 ) return;
-    MassCutoff *massCutoff = mw->getUserMassCutoff();
-
-    mw->fragPanel->clearTree();
-    int count=0;
-    for ( unsigned int i=0; i < samples.size(); i++ ) {
-        mzSample* sample = samples[i];
-
-        for (unsigned int j=0; j < sample->scans.size(); j++ ) {
-            Scan* scan = sample->scans[j];
 
-            if (scan->mslevel > 1 && scan->precursorMz >= mzmin && scan->precursorMz <= mzmax) {
+    //clear MS2 events list
+	mw->fragPanel->clearTree();
+    
+	int count = 0;
+    for (auto sample : samples) {
+        for (auto scan : sample->scans) {
+            if (scan->mslevel > 1 &&
+				scan->precursorMz >= mzmin &&
+				scan->precursorMz <= mzmax) {
 
                 mw->fragPanel->addScanItem(scan);
-                if (scan->rt < eicParameters->_slice.rtmin || scan->rt > eicParameters->_slice.rtmax) continue;
+                if (scan->rt < eicParameters->_slice.rtmin || 
+					scan->rt > eicParameters->_slice.rtmax) {
+					continue;
+				}
 
-        		QColor color = QColor::fromRgbF( sample->color[0], sample->color[1], sample->color[2], 1 );
-                EicPoint* p  = new EicPoint(toX(scan->rt), toY(10), NULL, getMainWindow());
+        		QColor color = QColor::fromRgbF(sample->color[0],
+												sample->color[1],
+												sample->color[2],
+												1);
+                EicPoint* p  = new EicPoint(toX(scan->rt),
+											toY(10),
+											NULL,
+											getMainWindow());
```

Comment:
The indentation here and [above](https://github.com/ElucidataInc/ElMaven/pull/996/commits/251f6f3d27efad5c7e9226a8e8663e3c02db3b5f#diff-999b8a9c0e6dc83ba91ee104baffd2c2R2012) looks a bit weird

Reply:
EicWidget is tab-indented so it looks weird in different editors. I can make them space-indented in the ui_fragSpectra branch though. To prevent merge conflicts with further branches.